mcp-npm/src/lib/config.ts

```diff
@@ -0,0 +1,63 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as os from 'os';
+
+interface PackageConfig {
+  jarVersion: string;
+  downloadUrl: string;
+}
+
+interface PackageJson {
+  config: PackageConfig;
+}
+
+export function getPackageConfig(): PackageConfig {
+  const packageJsonPath = path.join(__dirname, '..', '..', 'package.json');
+  const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, 'utf8')) as PackageJson;
+  return packageJson.config;
+}
```

Comment:
if config must be done this way, I think you can require/import the json file instead of reading it every time. Might mess up your tests since it gets cached.